.gitignore

```diff
@@ -3,7 +3,17 @@ results/*.csv
 results/*.json
 results/test/*.csv
 results/test/*.json
-results/test.*/county
+results/county/*
+
+# don't upload csvs/files
+counties.csv
+counties.dbf
+counties.shp
+counties.shx
+states.csv
+states.dbf
+states.shp
```

Comment:
we may have to delete the files since they are added to gitignore, right?

Reply:
yes yes